Hm... I get:
```
> Task :test FAILED
FAILURE: Build failed with an exception.
```
Not quite sure why, there is no exception. It it 
`INFO: Properties resource not found: class path resource [omero.properties] cannot be opened because it does not exist` ? Or because of `> No tests found for given includes: [integration.ImportAsTest.testImportAs](--tests filter)`?

--- a/components/tools/OmeroJava/test/integration/AbstractServerImportTest.java
+++ b/components/tools/OmeroJava/test/integration/AbstractServerImportTest.java
@@ -26,10 +26,14 @@ import java.util.List;
 import loci.formats.in.FakeReader;
 
 import ome.formats.OMEROMetadataStoreClient;
+import ome.formats.importer.ImportCandidates;
 import ome.formats.importer.ImportConfig;
 import ome.formats.importer.ImportContainer;
+import ome.formats.importer.ImportEvent;
 import ome.formats.importer.ImportLibrary;
 import ome.formats.importer.ImportLibrary.ImportCallback;
+import ome.formats.importer.IObservable;
+import ome.formats.importer.IObserver;
 import ome.formats.importer.OMEROWrapper;
 import ome.formats.importer.util.ProportionalTimeEstimatorImpl;
 import ome.formats.importer.util.TimeEstimator;
@@ -111,5 +115,46 @@ public class AbstractServerImportTest extends AbstractServerTest {
         cb.loop(60 * 60, 1000); // Wait 1 hr per step.
         Assert.assertNotNull(cb.getImportResponse());
         return req.location;
+    }
+
+    /**
+     * Returns the import candidates corresponding to the specified file.
+     *
+     * @param f
+     *            The file to handle.
+     * @return See above.
+     */
+    protected ImportCandidates getCandidates(File f)
+        throws Exception
+    {
+        ImportConfig config = new ImportConfig();
+        OMEROWrapper reader = new OMEROWrapper(config);
+        String[] paths = new String[1];
+        paths[0] = f.getAbsolutePath();
+        IObserver o = new IObserver() {
+            public void update(IObservable importLibrary, ImportEvent event) {
+
+            }
+        };
+        return new ImportCandidates(reader, paths, o);
+    }
+
+    /**
+     * Import the image with the specified file name
+     *
+     * @param name The name of the file
+     */
+    protected boolean importImageFile(String name)
+        throws Throwable
+    {
+        File f = File.createTempFile(name + ModelMockFactory.FORMATS[0], "."
+                + ModelMockFactory.FORMATS[0]);
+        mmFactory.createImageFile(f, ModelMockFactory.FORMATS[0]);
+        f.deleteOnExit();
+        ImportConfig config = new ImportConfig();
+        ImportLibrary library = new ImportLibrary(createImporter(), new OMEROWrapper(
+                config));
+        ImportCandidates candidates = getCandidates(f);
+        return library.importCandidates(config, candidates);
     }
 }

--- a/components/tools/OmeroJava/test/integration/AbstractServerTest.java
+++ b/components/tools/OmeroJava/test/integration/AbstractServerTest.java
@@ -2529,44 +2529,4 @@ public class AbstractServerTest extends AbstractTest {
         return newUser;
     }
 
-    /**
-     * Returns the import candidates corresponding to the specified file.
-     *
-     * @param f
-     *            The file to handle.
-     * @return See above.
-     */
-    protected ImportCandidates getCandidates(File f)
-        throws Exception
-    {
-        ImportConfig config = new ImportConfig();
-        OMEROWrapper reader = new OMEROWrapper(config);
-        String[] paths = new String[1];
-        paths[0] = f.getAbsolutePath();
-        IObserver o = new IObserver() {
-            public void update(IObservable importLibrary, ImportEvent event) {
-
-            }
-        };
-        return new ImportCandidates(reader, paths, o);
-    }
-
-    /**
-     * Import the image with the specified file name
-     *
-     * @param name The name of the file
-     */
-    protected boolean importImageFile(String name)
-        throws Throwable
-    {
-        File f = File.createTempFile(name + ModelMockFactory.FORMATS[0], "."
-                + ModelMockFactory.FORMATS[0]);
-        mmFactory.createImageFile(f, ModelMockFactory.FORMATS[0]);
-        f.deleteOnExit();
-        ImportConfig config = new ImportConfig();
-        ImportLibrary library = new ImportLibrary(createImporter(), new OMEROWrapper(
-                config));
-        ImportCandidates candidates = getCandidates(f);
-        return library.importCandidates(config, candidates);
-    }
 }

--- a/components/tools/OmeroJava/test/integration/ImportAsTest.java
+++ b/components/tools/OmeroJava/test/integration/ImportAsTest.java
@@ -38,7 +38,7 @@ import org.testng.annotations.Test;
  * @since 5.6.1
  */
 @Test(groups = { "import", "integration"})
-public class ImportAsTest extends AbstractServerTest {
+public class ImportAsTest extends AbstractServerImportTest {
 
 
     @DataProvider(name = "permission")

--- a/components/tools/OmeroJava/test/integration/ImportAsTest.java
+++ b/components/tools/OmeroJava/test/integration/ImportAsTest.java
@@ -20,7 +20,6 @@
 package integration;
 
 
-import omero.SecurityViolation;
 import omero.api.ISessionPrx;
 import omero.model.Session;
 import omero.sys.EventContext;

--- a/components/tools/OmeroJava/test/integration/ImportLibraryTest.java
+++ b/components/tools/OmeroJava/test/integration/ImportLibraryTest.java
@@ -73,7 +73,7 @@ import com.google.common.collect.ImmutableMap;
  * @since 5.0.0
  */
 @Test(groups = { "import", "integration", "fs" })
-public class ImportLibraryTest extends AbstractServerTest {
+public class ImportLibraryTest extends AbstractServerImportTest {
 
     /**
      * Tests the <code>ImportImage</code> method using an import container

